src/main/features/first-contact/guards/ClaimantRequestedCCJGuard.ts

```diff
@@ -0,0 +1,17 @@
+import * as express from 'express'
+import { ErrorPaths } from 'first-contact/paths'
+import { User } from 'idam/user'
+
+const logger = require('@hmcts/nodejs-logging').getLogger('first-contact/guards/ClaimantRequestedCCJGuard')
```

Comment:
don't use uppercase for the filename, it causes problems with case sensitive file systems and imports